### PR TITLE
Update github documentation including networks

### PR DESCRIPTION
**Pull request by jmmovilla**
@hmunfru @jframos

### CODE REVIEW

**Comment by flopezag** [comment]
LGTM

File: doc/user_guide.rst [hunk 1]
```diff
@@ -322,7 +322,13 @@ This operation lists the environments stored in the catalogue. The following exa
                     <initial_number_instances>1</initial_number_instances>
                     <maximum_number_instances>1</maximum_number_instances>
                     <minimum_number_instances>1</minimum_number_instances>
-                    <name>{tier-id}</name>               
+                    <name>{tier-id}</name>
+                    <networkDto>
```
Comment (jframos) [inline doc/user_guide.rst]: NetworkDto or NetworkDtos ?

Comment (jmmovilla) [inline doc/user_guide.rst]: So far, it is NetworkDTo

**Comment by jframos** [comment]
LGTM

**Pull request merged**